### PR TITLE
App: Update Monitoring page

### LINKED ISSUE #48394: Update Monitoring page
Add the notice that Monitoring is only available for our Enterprise solutions. Designs [here](https://www.figma.com/file/tbM21yr4k9gJNKkjNc0a4m/UX%3A-Messaging%2FFeedback-in-App?node-id=3%3A610&t=kHc5u9zhLqUYl8sD-0).

This is a part of [UX: Messaging/Feedback in App](https://github.com/sourcegraph/sourcegraph/issues/48273).

### PR DESCRIPTION
**Pull request by philipp-spiess**
Closes #48394

## Test plan

Screenshot 2023-03-06 at 17 36 24

## App preview:

- [Web](https://sg-web-ps-add-app-header-to-code.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by philipp-spiess** [comment]
Converting this to WIP to address some [feedback](https://sourcegraph.slack.com/archives/C04F9E7GUDP/p1677775360114129?thread_ts=1677764055.420189&cid=C04F9E7GUDP)

File: client/web/src/enterprise/code-monitoring/CodeMonitoringGettingStarted.tsx [hunk 1]
```diff
@@ -103,20 +104,21 @@ export const CodeMonitoringGettingStarted: React.FunctionComponent<
...
+                            to="https://about.sourcegraph.com"
+                            onClick={() =>
+                                eventLogger.log('ClickedOnEnterpriseCTA', { location: 'MonitoringGettingStarted' })
+                            }
+                        >
+                            get Sourcegraph Enterprise
+                        </Link>
+                        .
+                    </CallToActionBanner>
+                ))}
```
Comment (philipp-spiess) [inline client/web/src/enterprise/code-monitoring/CodeMonitoringGettingStarted.tsx:121]: @marekweb Do you think we want to do anything special here for app? What about UTM tags?

**Comment by philipp-spiess** [comment]
Updated the implementation, this is ready for a review 😎

**Review by slimsag — APPROVED** [review]

**Comment by almeidapaulooliveira** [comment]
"Cool CTA"? 😆

**Review by almeidapaulooliveira — APPROVED** [review]

**Pull request merged**